src/connectors/atomService.ts

```diff
@@ -218,6 +227,32 @@ export class AtomService {
     return record
   }
 
+  updateJsonColumn = async ({
+    table,
+    where,
+    jsonColumn = 'extra',
```

Comment:
is `extra` a de facto standard for a extra JSON column in a table? otherwise we shouldn't make it as default column here?